### PR TITLE
Update CODE_OF_CONDUCT.md

### PR DESCRIPTION
**Pull request by tanya-vedi**

### CODE REVIEW

**Comment by tanya-vedi** [comment]
#498

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -1 +1 @@
-This project is governed by the Contributor Covenant version 1.4 (https://www.contributor-covenant.org/version/1/4/code-of-conduct.html). All contributors and participants agree to abide by its terms. To report violations, send an email to covenant@idolhands.com.
+This project is governed by the [Contributor Covenant version 1.4] (https://www.contributor-covenant.org/version/1/4/code-of-conduct.html). All contributors and participants agree to abide by its terms. To report violations, send an email to covenant@idolhands.com.
```
Comment (ianbollinger) [inline CODE_OF_CONDUCT.md]: The link appears to be broken on GitHub's markdown renderer. While I think the original Markdown implementation allowed a space between the `]` and `(` characters in a link, both GitHub Flavored Markdown and CommonMark seem to disallow it. Anyway, that was probably more verbiage than an extraneous space warranted, sorry.

**Review by jordan2936 — CHANGES_REQUESTED** [review]
Parts Need to delete

File: CODE_OF_CONDUCT.md [hunk 2]
```diff
@@ -1 +1 @@
-This project is governed by the Contributor Covenant version 1.4 (https://www.contributor-covenant.org/version/1/4/code-of-conduct.html). All contributors and participants agree to abide by its terms. To report violations, send an email to covenant@idolhands.com.
```
Comment (jordan2936) [inline CODE_OF_CONDUCT.md]: Delete

**Review by jordan2936 — APPROVED** [review]

**Pull request closed without merging**